app/services/analytics.rb

```diff
@@ -229,6 +229,8 @@ def browser_attributes
   PROOFING_DOCUMENT_TIMEOUT = 'Proofing Document Timeout'.freeze
   PROOFING_RESOLUTION_TIMEOUT = 'Proofing Resolution Timeout'.freeze
   RATE_LIMIT_TRIGGERED = 'Rate Limit Triggered'.freeze
+  REPORT_SP_USER_COUNTS = 'Report SP User Counts'.freeze
+  REPORT_TOTAL_SP_USER_COUNTS = 'Report Total SP User Counts'.freeze
```

Comment:
this is now the total count across all SPs, I don't think it should be considered an SP User Count
```suggestion
  REPORT_TOTAL_USER_COUNTS = 'Report Total User Counts'.freeze
```

Reply:
hmmm...needs to be a way to delineate from User.count and totally registered user count (which includes non sp linked users).  Any other ideas?

Reply:
sounds like we're going to have separate events for User.count and total registered user count, so maybe we just don't include this at all? I don't see what benefit this gives us because it would double-count users at multiple SPs?

Reply:
well now...excellent point!... scratch this

Reply:
Great catch

Reply:
It needs a separate query for unique users in the table.  Rather than do that the one we normally report is fully registered users.  I'll do that.

Reply:
Oops.  can't ...then we don't have breakdown... new query it is

Reply:
3 queries actually...

Reply:
we can start reporting cool things like percentage reused etc. too